edx_sga/sga.py

```diff
@@ -706,6 +733,13 @@ def file_storage_path(self, sha1, filename):
         return path
 
 
+def is_finalized_submission(submission_data):
+    """
+    Helper function to determine whether or not a Submission was finalized by the student
+    """
+    return submission_data['answer'].get('finalized', True)
```

Comment:
@gsidebo and I discussed this and this seemed better than the alternative -- requiring every learner to revisit the XBlock and click the submit button. 

This won't be a problem for our Residential MITx release because there won't be any courses in progress when we upgrade. But we'll have to call this out in the README so that anyone else upgrading will know.

Reply:
i'll add some logic that checks for those conditions with `submission_data`